### ISSUE TITLE
Unsigned themes do not fully apply

### ISSUE THREAD

**Issue by StraggleCraft** [AutoDarkMode/Windows-Auto-Night-Mode#778]
### Description

Unsigned themes do not fully apply when using ADM.  I see that ADM uses SecureUXTheme, so I would have thought it would be possible to use themes that I've patched using ADM...  I'm not sure if this is a bug or a feature request.   The logs don't show anything interesting here, so I figured I'd see if this was a feature that could be added, or if it is a bug.

### Log Data

2023-09-25 01:07:24 | Info | MessageParser.Parse: signal received: invoke theme switch request 
2023-09-25 01:07:41 | Info | PostponeItem.StartExpiry: PauseAutoSwitch had its expiry at 25.09.2023 06:00:01 cancelled 
2023-09-25 01:07:49 | Info | ComponentManager.GetComponentsToUpdate: components queued for update: [OfficeSwitch] 
2023-09-25 01:07:49 | Info | OfficeSwitch.HandleSwitch: update info - previous: Light, now: Dark, mode: Switch 
2023-09-25 01:07:49 | Info | ThemeManager.UpdateTheme: dwm management: full refresh requested by user 
2023-09-25 01:08:05 | Info | Tm2Handler.SetTheme: applied theme ADMUnmanagedDark, from origin: C:\Users\User\AppData\Local\Microsoft\Windows\Themes\Penumbra10s.theme directly via IThemeManager2 
2023-09-25 01:08:05 | Warn | ThemeHandler.ApplyIThemeManager2: theme switching took longer than expected (15.7562201 seconds) 
2023-09-25 01:08:05 | Info | ThemeManager.UpdateTheme: dark theme switch performed, source: Manual 
2023-09-25 01:08:05 | Info | Service.ToggleTheme: ui signal received: theme toggle: switching to dark theme 

### Commit Hash, Version and Windows Build

- Commit: `57ef363`
- Service/App: `10.4.0.35`
- Updater: `3.1.4`
- Shell: `1.3.3.0`
- .Net: `7.0.5`
- Windows: `19045.3086`

### Screenshots / Videos

![image](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/assets/10663364/a7617acd-d81a-472f-8d4c-78ea104ea81f)
![image](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/assets/10663364/cb5ff260-9f80-4633-97f4-01d7e0ac62bd)

**Comment by Spiritreader** [comment]
We know that, and it's currently not supported because it would require full integration of https://github.com/namazso/SecureUxTheme

We do not use SecureUXTheme per se, just the raw windows API that namaszo provided to us.

**Comment by StraggleCraft** [comment]
I saw that "automation" was possible with more recent builds of SecureUxTheme via a C FFI Interface... allegedly being possible to do with powershell etc...
This is way over my head though, Does this at all make the task easier than building a new SecureUX library? or still out of scope, too much work.  
Thanks for the attention!

https://github.com/namazso/SecureUxTheme/issues/86
https://github.com/namazso/SecureUxTheme/issues/120

**Comment by Spiritreader** [comment]
> I saw that "automation" was possible with more recent builds of SecureUxTheme via a C FFI Interface.

Yes I compiled a version of the c ffi with and without signature verification to be used with the ADM bridge, but that was retired in favor of the in-built solution that didn't require external dlls
https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/blob/master/IThemeManager2Bridge/ThemeDllWrapper.cs

> https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/blob/master/IThemeManager2Bridge/ThemeDllWrapper.cs

I can do that, although themes with fake signatures were never meant to be supported in the first place.

**Comment by Jay-o-Way** [comment]
Is this still an issue?

**Comment by Spiritreader** [comment]
This is now a candidate for release version 11 or 11.1. I don't know if we can get it out in time, but there's now a possibility to force window refreshes.

I will push an update later today that should in theory fix this. I'll notify here when the beta is out and it would be awesome if you could test @StraggleCraft

**Comment by Spiritreader** [comment]
Build 11.0.0.12 is released and should work!
